.github/workflows/ci-syntax-tests.yml

```diff
@@ -3,16 +3,12 @@ name: CI Syntax Tests
 on:
   push:
     branches:
-      - '**'
-    tags-ignore:
-      - '**'
```

Comment:
I'd suggest to keep `tags-ignore` to avoid another CI run triggered by creating tags/releases.

Reply:
I could see situations where the exact commit that a tag was created for was never pushed as a branch HEAD before and realistically the tag is what you are most interested in to pass tests, so I think we should keep tag pushes as triggers imo.

Reply:
I'd suggest to always create tags only on existing main branches:

st4107 branch -> 4107-x.x.x
main branch -> 4126-x.x.x

I've already seen people pushing tags anywhere in nowhere, but I find that bad practice.

Reply:
Well, let's stick with it for now. I'm also curious to see how it turns out in practice, e.g. if the action run is linked on the tag page etc. If it turns out to be annoying (which I doubt), we can adjust it later.